core/controllers/question_editor.py

```diff
@@ -46,12 +46,11 @@ def post(self, skill_id):
                 (question_dict['id'] is not None) or
                 ('question_state_data' not in question_dict) or
                 ('language_code' not in question_dict) or
-                (question_dict['version'] != 1) or
-                (question_dict['question_state_schema_version'] != (
-                    feconf.CURRENT_STATES_SCHEMA_VERSION))):
+                (question_dict['version'] != 1)):
             raise self.InvalidInputException
 
-
+        question_dict['question_state_schema_version'] = (
```

Comment:
Let's say the incoming question doesn't actually conform to the latest states schema version. What happens?

Important to add a backend integration test to ensure that an error does actually get raised.

Reply:
Done